htdocs/api/class/api_documents.class.php

```diff
@@ -601,7 +601,7 @@ public function getDocumentsListByElement($modulepart, $id = 0, $ref = '', $sort
 				throw new RestException(404, 'Event not found');
 			}
 
-			$upload_dir = getMultidirOutput($object) . '/'.dol_sanitizeFileName($object->ref);
+			$upload_dir = $conf->agenda->multidir_output[$object->entity ?? $conf->entity].'/'.get_exdir(0, 0, 0, 1, $object, 'agenda');
```

Comment:
Is it possible to fix inside the getMultidirOutput ?
Because future is to replace all things like 
`$conf->agenda->multidir_output[$object->entity ?? $conf->entity].'/'.get_exdir(0, 0, 0, 1, $object, 'agenda');`
into
`getMultidirOutput($object) . '/'.dol_sanitizeFileName($object->ref);`